CMakeLists.txt

```diff
@@ -260,6 +260,7 @@ if(BUILD_TOOLS AND WIN32)
         Microsoft::DirectXMesh
         Microsoft::DirectXTex
         Microsoft::DirectXMesh::Utilities)
+    target_link_options(uvatlastool PRIVATE -municode)
```

Comment:
This should have already been handled down on line 347. It's not properly guarded here.

Reply:
Did you forget to submit this change?

Reply:
This is still adding a `-municode` to all cases, not just Clang/MinGW.

Reply:
I think you just need to remove line 263 now that you added the changes below.

Reply:
Fixed in the PR.